app.py

```
@@ -0,0 +1,249 @@
+    tf_values = {}
+
+    defaults = {
+        "1h": {"ema9": 185.50, "ema21": 184.00, "ema50": 182.00, "rsi14": 62.0,
+               "macd_val": 0.50, "macd_sig": 0.05, "macd_hist": 0.45, "atr14": 1.20},
+        "15m": {"ema9": 185.20, "ema21": 184.80, "ema50": 0.0, "rsi14": 58.0,
+                "macd_val": 0.20, "macd_sig": 0.05, "macd_hist": 0.15, "atr14": 0.60},
+        "1m": {"ema9": 185.05, "ema21": 184.95, "ema50": 0.0, "rsi14": 42.0,
+               "macd_val": 0.03, "macd_sig": 0.01, "macd_hist": 0.02, "atr14": 0.10},
+    }
+
+    for tab, tf_key in zip(tf_tabs, ["1h", "15m", "1m"]):
```

Ruff (0.14.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)